src/parser/parse_redirect_utils.c

```diff
@@ -41,7 +42,7 @@ int	count_consecutive_spaces(const char *str)
 	int	i;
 
 	i = 0;
-	while (str && str[i] && str[i] == SPACE)
+	while (str && str[i] && str[i] == SPACE_CHAR)
 		i++;
```

Comment:
is this part necessary? Because then you will replace the spaces with spaces, correct?
if we don't remove it, could you change to ++i so it is the same as in line 36?

Reply:
It does indeed count spaces and then replace them with spaces, replacing with spaces happens linearly so for example:
 "  < {spaces_here} infile  |" the spaces in between need to be counted for the true length. (Looks like github removes extra white space).
 
 Replaced with ++i;